### DISCUSSION TITLE
Did you turn one time donation into a monthly subscription?

### DISCUSSION THREAD

**Discussion by Homero-79** [devlikeapro/waha#388] [General]
First of all thank you for creating this amazing project.
On your website you don't say that plus costs 19$ per month rather just 19$, and in other discussions you say it's a donation. However in your donation portal it says subscribe for 20$ per month. 
Just wondering.

**Comment by devlikepro** [comment]
**It doesn't require monthly subscriptions, once installed on your server - it always works!** (until WhatsApp made backward-incompatible changes, and you have to update the image, or you need to reinstall it on new server )

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
